pkg/commands/dictionary/dictionary_test.go

```diff
@@ -324,12 +324,16 @@ func describeDictionaryOKDeleted(i *fastly.GetDictionaryInput) (*fastly.Dictiona
 }
 
 func createDictionaryOK(i *fastly.CreateDictionaryInput) (*fastly.Dictionary, error) {
+	// This is a bit nastly, but just an artifact of testing- nil is false.
```

Comment:
I think it's fine not to have the comment here.
```suggestion
```

Reply:
Cool: https://github.com/fastly/cli/pull/707/commits/a8210d4bb2b90b9427b452f229817faa14326835